pywps/inout/inputs.py

```diff
@@ -313,7 +313,10 @@ def from_json(cls, json_input):
             elif allowed_value['type'] == 'novalue':
                 allowed_values.append(NoValue())
             elif allowed_value['type'] == 'valuesreference':
-                allowed_values.append(ValuesReference())
+                allowed_values.append(ValuesReference(
```

Comment:
I guess you could do : `allowed_values.append(ValuesReference.from_json(allowed_value))`

Reply:
done. Also for `AllowedValues`.